stream-chat-android-client/src/main/java/io/getstream/chat/android/client/socket/ChatSocket.kt

```diff
@@ -297,9 +295,7 @@ internal open class ChatSocket constructor(
 
     private fun callListeners(call: (SocketListener) -> Unit) {
         synchronized(listeners) {
-            listeners.forEach { listener ->
-                eventUiHandler.post { call(listener) }
-            }
+            listeners.forEach(call)
         }
```

Comment:
This might be a breaking change. In other words, this may affect those who use `ChatClient.subscribeFor...`. Customers may expect those events coming within the `Main` thread